packages/plugin-session-replay-react-native/src/session-replay.ts

```diff
@@ -40,6 +40,7 @@ export class SessionReplayPlugin implements EnrichmentPlugin<ReactNativeClient,
       this.sessionReplayConfig.sampleRate ?? 1,
       this.sessionReplayConfig.enableRemoteConfig ?? true,
       this.sessionReplayConfig.logLevel ?? LogLevel.Warn,
+      this.sessionReplayConfig.autoStart,
```

Comment:
should fall back to true
```suggestion
      this.sessionReplayConfig.autoStart ?? true,
```